dags.yaml

```diff
@@ -272,7 +285,13 @@ bqetl_fenix_event_rollup:
     retry_delay: 30m
 
 bqetl_stripe:
-  schedule_interval: daily
+  schedule_interval: 30 0 * * *
+  description: |
+    Daily derived tables on top of data imported from Stripe.
+
+    Depends on the `stripe` DAG which starts at midnight UTC;
+    we allow 30 minutes for that DAG to complete before this one
+    is scheduled to start.
```

Comment:
cc @relud on this Stripe scheduling change.

Reply:
the timing lgtm, but due to #1920 may break this dag

Reply:
#1921 provides a workaround for this dag until #1920 can be fixed.